docs/Plugins/sonarqube.md

```diff
@@ -9,6 +9,9 @@ description: >
 This plugin collects SonarQube data through its REST APIs. SonarQube is a tool for static code analysis and code quality management. It can help you discover potential problems and defects in your code, and provide suggestions and solutions.
 (Please note that this version of SonarQube cannot collect issues with more than 10k+)
 
+## Supported Versions
+Available for v8.x, v9.x
```

Comment:
Available for SonarQube Server v8.x, v9.x. Check [this doc](relative url of the supported data sources) for more details.